<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><!-- InstanceBegin template="/Templates/eebmaxcontent.dwt.php" codeOutsideHTMLIsLocked="false" --><meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"><title>ITS Secondary Structure</title><link href="../etc/style.css" rel="stylesheet" type="text/css"><link href="../etc/eebglobal.css" rel="stylesheet" type="text/css"><!-- InstanceBeginEditable name="StyleArea" --><style type="text/css"></style><!-- InstanceEndEditable --><script type="text/javascript">  var _gaq = _gaq || [];  _gaq.push(['_setAccount', 'UA-19439610-1']);  _gaq.push(['_trackPageview']);  (function() {    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);  })();</script></head><body><table border="0" cellpadding="0" cellspacing="0" width="100%">		<!--DWLayoutTable--><tbody><tr>	<td colspan="7" class="border_bottom" style="white-space: nowrap;" align="left" bgcolor="#000066" height="80" valign="top">	<a href="http://www.uconn.edu/">	<img src="../etc/blue_top.gif" alt="UConn Home" border="0" height="78" width="250"></a>	<a href="http://hydrodictyon.eeb.uconn.edu/eebwww/"><img src="../etc/banner.jpg" alt="Banner" border="0" height="78" width="600"></a>	</td>	</tr></tbody></table></td></tr><tr><td valign="top"><table border="0" cellpadding="0" cellspacing="0"><!--DWLayoutTable--><tbody><tr>	<td class="global_nav_left_01" valign="top" width="162">	<table border="0" cellpadding="0" cellspacing="0">	<tbody><tr>	<td align="left" valign="top" width="162">	<a href="../index.htm" class="global_left_nav_2top">Nicholas P. Tippery</a>	<ul class="sublink">		<li class="sublink"><a href="../index.htm" class="link_3">Research Home</a></li>		<li class="sublink"><a href="../Tippery-CV.htm" class="link_3">Curriculum Vitae</a></li>	</ul>	<a href="../Meny-taxonomy/index.htm" class="global_left_nav2">Menyanthaceae</a>	<ul class="sublink">		<li class="sublink"><a href="../morphology/index.htm" class="link_3">Morphology</a></li>		<li class="sublink"><a href="../phylogeny/index.htm" class="link_3">Phylogeny</a></li>		<li class="sublink"><a href="../Meny-taxonomy/index.htm" class="link_3">Taxonomy</a></li>	</ul>	<a href="../ITS-secondary/index.htm" class="global_left_nav2">ITS Secondary Structure</a>	<a href="http://www.eeb.uconn.edu/people/les/" class="global_left_nav2">Les Lab</a></td></tr></tbody></table></td><td align="left" bgcolor="#eeeeee" valign="top"><table align="center" border="0" cellpadding="0" cellspacing="0"><!--DWLayoutTable--><tbody><tr><td style="padding: 20px;" align="left" valign="top"><!-- InstanceBeginEditable name="ContentArea" --><div id="pagecontent"><span class="category">ITS Secondary Structure</span></p><p><table border=0 cellspacing=0><tr><td valign=top><p class="mailingaddr">&nbsp;&nbsp;&nbsp;&nbsp;The internal transcribed spacer (ITS) refers to two regions of DNA that are interspersed with the 18S, 5.8S, and 26S ribosomal RNA (rRNA) genes. The rRNA genes are strongly conserved for their function in protein synthesis (which depends on the secondary structure of their single-stranded RNA transcript). The spacer regions are much more variable and originally had no known function. Recent research, however, has determined that the ITS regions in fact function to cleave the rRNA genes from the RNA transcript of the whole ITS/rRNA region. The function of ITS, like the rRNA genes, depends on certain aspects of secondary structure, which are remarkably conserved across broad phylogenetic distances.<br>&nbsp;&nbsp;&nbsp;&nbsp;Don Les and I have described a method to use secondary structure characteristics in a phylogenetic context to group taxa that have more similar ITS secondary structures. In an examination of Menyanthaceae taxa, we were able to recover a phylogenetic tree very similar to the tree that was obtained using nucleotide data in a more traditional phylogenetic analysis.<br><img height=100 align=top src="ITS-region.gif" alt="Schematic of ITS and rRNA region." border=0><p class="mailingaddr">&nbsp;&nbsp;&nbsp;&nbsp;The method and results detailed below are published in:<br>Tippery, N.P. and D.H. Les. 2008. Phylogenetic analysis of the internal transcribed spacer (ITS) region in Menyanthaceae using predicted secondary structure. <i>Molecular Phylogenetics and Evolution</i> 49: 526-537. [ <a href="http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WNH-4T542DX-2&_user=669286&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_version=1&_urlVersion=0&_userid=669286&md5=64bf912d108ae713a64a0b9bcec97599" target="_blank">abstract</a> ]</p><tr><td valign=top><p class="mailingaddr"><b>ITS Subregions</b><br>&nbsp;&nbsp;&nbsp;&nbsp;Seven stem subregions of ITS, four in ITS-1 and three in ITS-2, are strongly conserved within Menyanthaceae and correspond to similarly conserved regions in the related family Asteraceae. The illustration below depicts likely structures for the ITS-1 and ITS-2 regions of <i>Nymphoides cordata</i>, which were predictively modeled using an algorithm to minimize free energy. The structure of 5.8S, which is located between ITS-1 and ITS-2, is not shown.<tr><td align=left valign=top><p class="mailingaddr"><img height=300 align=left src="Fig1_Ncor02-ITS_07-1009.gif" alt="Secondary structure of ITS-1 and ITS-2 from Nymphoides cordata." border=0><tr><td valign=top><p><p class="mailingaddr"><b>Encoding Data</b><br>&nbsp;&nbsp;&nbsp;&nbsp;In order to obtain data that can be used in phylogenetic analyses, the seven subregions were coded independently. The figure below depicts the data encoding process comparatively across three species. Nucleotides that form complementary base-pairs were coded with '1', and nucleotides without a complementary pair (either a mismatch or a gap) were given a '0'. The data were aligned among taxa to maximize structural and nucleotide sequence similarity. Gaps that resulted from alignment were encoded separately.<tr><td align=left valign=top><p class="mailingaddr"><img height=300 align=left src="Fig2_07-0730.gif" alt="Example of ITS secondary structure data encoding in three Menyanthaceae species." border=0><tr><td valign=top><p><p class="mailingaddr"><b>Phylogenetic Analyses</b><br>&nbsp;&nbsp;&nbsp;&nbsp;The encoded and aligned data can be analyzed using phylogenetic software under a variety of models. The trees below represent trees from a maximum parsimony analysis. In each figure, the left-hand tree depicts the strict-consensus parsimony tree, and nodal support values are given for parsimony bootstrap (above) and Bayesian posterior probability (below). The right-hand tree in each figure shows the corresponding phylogram of one of the most-parsimonious trees. For comparison, the left-hand figure below depicts the tree derived from traditional analysis of aligned ITS nucleotide sequence data. The right-hand figure shows the tree obtained by analyzing ITS secondary structure data. Nodes with letter labels represent the same clades in both trees.</table><table border=0 cellspacing=0><tr><td align=left valign=top><p class="mailingaddr"><img height=400 align=left src="Fig3-nuc_08-0725.gif" alt="Phylogeny derived from analysis of aligned ITS nucleotide sequences." border=0><br>Phylogeny derived from analysis of aligned ITS nucleotide sequences.<td valign=top><p class="mailingaddr"><img height=400 align=left src="Fig4-str_08-0725.gif" alt="Phylogeny derived from analysis of aligned ITS secondary structure data." border=0><br>Phylogeny derived from analysis of aligned ITS secondary structure data.</table><table border=0 cellspacing=0><tr><td valign=top><p><p class="mailingaddr"><b>Ancestral State Reconstruction</b><br>&nbsp;&nbsp;&nbsp;&nbsp;The ITS secondary structure data can be used additionally to infer ancestral states for the structure of ITS subregions, and thereby to investigate the mechanisms and constraints of evolution in the ITS-1 and ITS-2 RNA molecules. Data analyzed in this manner may help to elucidate functions of the ITS transcript that have not yet been identified. The figure below shows the inferred ancestral states for subregion 1A of ITS-1, using the same node labels as the trees above. The state for ancestor B was omitted because of the uncertain resolution of that node (see trees above). Boxed nucleotides highlight changes from the most immediate ancestor (the nearest node to the left; e.g., changes shown for ancestor F are relative to ancestor G).<tr><td align=left valign=top><p class="mailingaddr"><img width=400 align=left src="Fig5-1A_08-0725.gif" alt="Secondary structure of ITS-1 and ITS-2 from Nymphoides cordata." border=0></table><!-- InstanceEndEditable -->
</td>
</tr>
</tbody></table>
</td>
</tr>
</tbody></table>
</td>
</tr>
<tr>
<td height="62" valign="top">

<table style="height: 62px;" border="0" cellpadding="0" cellspacing="0" width="100%">
<!--DWLayoutTable-->
				<tbody><tr>
					<td class="border_top" width="73%">
					<td class="border_top" align="left" valign="middle" width="27%">
						<span class="footer_grey10px">Copyright &copy; 2010 Nicholas P. Tippery<br>
						75 N. Eagleville Road, Unit 3043<br>
						Storrs, Connecticut 06269-3043, U.S.A.<br>
						Tel: +01 860 486-3937 &nbsp; Fax: +01 860 486-6364
						</span>
					</td>
				</tr>
			</tbody></table>
		</td>
	</tr>
</tbody></table>
<!-- InstanceEnd --></body></html>